src/Components/Activities/Activities.js

```diff
@@ -25,7 +26,7 @@ const Activities = ({ activities, updateActivity }) => {
         <h2>{activities[currentActivityIndex].activity}</h2>
         {currentActivityIndex < activities.length - 1 && <button onClick={() => nextActivity()}>→</button>}
       </div>
-      <button id="go-button" onClick={() => selectActivity()}>Let's do it!</button>
+      <Link to="/why-are-you-feeling-that-way" className="go-button" onClick={() => selectActivity()}>Let's do it!</Link>
```

Comment:
Beautiful! FYI, I have made the updates to this file that y'all suggested and will push that up with the Journal prompt component later tonight!